webdriver/path-test.js

```diff
@@ -15,14 +15,17 @@ exports.tests = function(ctx) {
       const page = await ctx.browser.newPage();
       const url = `${ctx.server.url}/${view}/2dcontext/building-paths`;
       await page.goto(url);
-      await page.waitForSelector(`wpt-${view}`);
+      await page.waitForFunction((view) => {
+        const results = document.querySelector(`wpt-${view}`);
+        return !!results && !results.isLoading;
+      }, {}, view);
       const linkNames = await page.$eval(
         `wpt-${view}`,
         results => {
           return Array.from(results.shadowRoot.querySelectorAll('path-part'))
               .map(p => p.shadowRoot.querySelector('a').innerText.trim());
         })
-      expect(linkNames).to.equal([
+      expect(linkNames).to.eql([
```

Comment:
```suggestion
      expect(linkNames).to.deep.equal([
```

`eql` is a bit too magical / not very readable (I would think it's a shorthand of `equal` if I didn't see this change).